projects/views.py

```diff
@@ -259,7 +279,22 @@ def delete_flavor(request, project_slug):
         pk = request.POST.get("flavor_pk")
         # TODO: Check that the user has permission to delete this flavor.
         flavor = Flavor.objects.get(pk=pk, project=project)
-        flavor.delete()
+
+        can_flavor_be_deleted = True
```

Comment:
Same here, it would be nice with a refactor of the logic so it becomes more testable and with a unit test on this.

Reply:
I will take this commit out into a separate PR.